pkg/k8sutil/ceph_rbd_commands.go

```diff
@@ -0,0 +1,145 @@
+/*
```

Comment:
Is there something specific to rbd in this file? Could we name it exec.go?

Reply:
done